loadable_apps/loadable_sample/wifiapp/binary_update.c

```diff
@@ -412,7 +412,7 @@ static void binary_update_same_version_test(void)
 	binary_update_getinfo(APP_NAME, &bin_info);
 
 	/* Try to create binary file with old version */
-	ret = binary_manager_open_new_entry(APP_NAME, bin_info.version);
+	ret = binary_manager_open_new_entry(APP_NAME, (int)bin_info.version);
```

Comment:
There is no int type version. We have uint32_t or float. How about changing the `binary_manager_open_new_entry`'s param type one of uint32_t or float?